yb-voyager/cmd/importDataFileTaskImporter.go

```diff
@@ -56,15 +60,27 @@ func NewFileTaskImporter(task *ImportFileTask, state *ImportDataState, workerPoo
 	progressReporter.ImportFileStarted(task, totalProgressAmount)
 	currentProgressAmount := getImportedProgressAmount(task, state)
 	progressReporter.AddProgressAmount(task, currentProgressAmount)
+	isTableColocated := false
+
+	if useColocatedImportBatchQueue {
+		yb, ok := tdb.(*tgtdb.TargetYugabyteDB)
+		if !ok {
+			return nil, fmt.Errorf("tdb is not of type TargetYugabyteDB. Cannot use colocated import batch queue")
+		}
+		isTableColocated, err = yb.IsTableColocated(task.TableNameTup)
```

Comment:
We are already calculating this in the picker; can we use that somehow without going to DB again?